app/models/collection_item.rb

```diff
@@ -191,12 +191,10 @@ def send_work_invitation
 
   after_update :notify_of_status_change
   def notify_of_status_change
-    if saved_change_to_unrevealed? && item.respond_to?(:new_recipients)
-      # making sure notify_recipients in the work model has not already notified
-      if item.new_recipients.present?
-        notify_of_reveal
-      end
-    end
+    return unless saved_change_to_unrevealed? && item.respond_to?(:new_gifts)
```

Comment:
Should this method still be here, per @tickinginstant's [comment on the old PR](https://github.com/otwcode/otwarchive/pull/3981#discussion_r668238836)?

Reply:
I cannot remember why I didn't take it out, so I'll try it and we can see what happens.